server/index.js

```diff
@@ -103,8 +103,33 @@ app.put('/api/edit-reservation', (req, res, next) => {
   const params = [partySize, uniqueCode];
   db.query(sql, params)
     .then(result => {
-      const update = result.rows[0];
-      res.status(200).json(update);
+      if (result.rowCount === 1) {
+        const update = result.rows[0];
+        res.status(201).json(update);
+      } else {
+        res.sendStatus(404);
+      }
+    })
+    .catch(err => next(err));
+});
+
+app.delete('/api/delete-reservation', (req, res, next) => {
+  const { uniqueCode } = req.body;
+  const sql = `
+  delete from "reservations"
+  where "uniqueCode" = $1;
+  `;
+  const params = [uniqueCode];
+  db.query(sql, params)
+    .then(result => {
+      if (result.rowCount === 1) {
+        const response = {
+          Success: 'requested reservation deleted'
+        };
+        return res.status(204).json(response);
```

Comment:
Express ignores response bodies if you use a `204` status code. Will `sendStatus(204)` with no JSON work for you here?